modules/cluster-logging-upgrading-logging-from-44.adoc

```diff
@@ -0,0 +1,304 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-upgrading.adoc
+
+[id="cluster-logging-updating-logging-from-44_{context}"]
+= Updating cluster logging from 4.4
+
+After updating the {product-title} cluster, you can update cluster logging from 4.4 to 4.6 by changing the subscription for the Elasticsearch Operator and the Cluster Logging Operator.
+
+When your cluster logging version is 4.4, you must update cluster logging as follows due to changes in the way that OLM provides operator packages.
+
+Additionally, it is recommended that you update Cluster Logging Operator before you update Elasticsearch Operator to avoid
+the need to restart the each Elasticsearch pod manually, one by one.
+
+Perform each update version by version in the following order:
+
+. Update Cluster Logging Operator to the latest version of 4.4, then update to 4.5.
+. Update Cluster Logging Operator to latest version of 4.5, then update to 4.6.
+. Update Elasticsearch Operator to the latest version of 4.4, then update to 4.5.
+. Update Elasticsearch Operator to latest version of 4.5, then update to 4.6
+
+.Procedure
+
+. Update the Cluster Logging Operator:
+
+.. From the OpenShift web console, click *Operators* -> *Installed Operators*.
+
+.. Select the `openshift-logging` project.
+
+.. Click *Cluster Logging*.
+
+.. Click *Subscription* -> *Channel*.
+
+.. In the *Change Subscription Update Channel* window, select *4.6*.
+
+.. On the *YAML* tab, set the retention policy in the `clusterlogging` custom resource (CR) as follows. You must specify the length of time that Elasticsearch should retain each log source. Enter an integer for the number of days, for example, `7d` for seven days. The `application` `maxAge` can be less than or equal to `7d`. The `infra` `maxAge` and `audit` `maxAge` must be `0d`, and you must *explicitly set this value* to override the default. Logs older than the `maxAge` are deleted. You must specify a retention policy for each log source or the Elasticsearch indices will not be created for that source.
++
+[source,terminal]
+----
+kind: "ClusterLogging"
+metadata:
+  name: "instance"
+  namespace: "openshift-logging"
+spec:
+  managementState: "Managed"
+  logStore:
+    type: "elasticsearch"
+    retentionPolicy:
+      application:
+        maxAge: 7d
+      infra:
+        maxAge: 1h
```

Comment:
Do you want the example to show the expected values? Change infra maxAge and audit maxAge values to 0d?

Reply:
No, these values are correct, as requested by the SME.

Reply:
@NautiluX can you double check which values we put in the webhook please?